Documentation for cholesky localization (#46)
pyscf/pyscf.github.io: source/user/lo.rst

ER localization, however, is computationally more expensive than the Boys or PM approaches.

Localized orbitals can be calculated via the pivoted Cholesky factorization of a density-like
matrix :math:`\mathbf{D} = \mathbf{C} \mathbf{C}^\dagger`. :cite:`Aquilante06` Since :math:`\mathbf{C}` is 
generally a rectangular matrix containing only the subset of :math:`N` orbitals intended for localization,
the matrix :math:`\mathbf{D}` is positive-semidefinite. It can be factored using a Cholesky decomposition
with full column pivoting,

.. math::
    \mathbf{P}^\dagger \mathbf{D} \mathbf{P} = \mathbf{L} \mathbf{L}^\dagger ,

where :math:`\mathbf{L}` is a lower triangular matrix and :math:`\mathbf{P}` is a permutation matrix.
In the end, the :math:`N` leftmost columns of :math:`\mathbf{P L}` are taken as the localized orbitals.

> While Cholesky orbitals are usually less well localized than, for example, PM or Boys orbitals, the procedure

Reviewer: ```suggestion
While Cholesky orbitals are usually not as localized as, for example, PM or Boys orbitals, the procedure
```
